prime-router/metadata/fhir_mapping/hl7/datatype/HDToExtensionAssigningAuthority.yml

```diff
@@ -0,0 +1,57 @@
+url:
+  type: SYSTEM_URL
+  value: 'assigning-authority'
+
+extension_1-namespaceId:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: namespace-id
+    valueString:
+      type: STRING
+      valueOf: HD.1
+      expressionType: HL7Spec
+
+extension_2-universalId:
+  generateList: true
+  condition: $universalIdType EQUALS ISO || $universalIdType EQUALS UUID
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: universal-id
+    valueString:
+      type: String
+      valueOf: HD.2
+      expressionType: HL7Spec
+  vars:
+    universalIdType: String, HD.3
+
+extension_3-universalIdType:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: universal-id-type
+    valueCode:
+      type: STRING
+      valueOf: HD.3
+      expressionType: HL7Spec
+
+extension_4-universalIdUnknownType:
```

Comment:
The mapping says to not include HD.2 if it is not ISO or UUID, so I capture in this extension so it can get mapped back to HL7

Reply:
It'd be nice if we could combine this with extension_5 below, but I know the boolean logic handling in this library can be finnicky. If we can't combine them, I'd at least recommend renaming the extensions to emphasize the distinction, maybe `universalIdUnknownType`/`universalIdNullType`.